core/chains/evm/chain_set.go

```diff
@@ -480,7 +480,7 @@ func NewTOMLChainSet(ctx context.Context, opts ChainSetOpts, chains []*v2.EVMCon
 	}
 	var enabled []*v2.EVMConfig
 	for i := range chains {
-		if e := chains[i].Enabled; e != nil && *e {
```

Comment:
This indicates that chain.Enabled == null means chain is disabled.

Reply:
For e== nil, this behavior is changing with this PR.
Just observing, not sure if practically it will be applicable.